code/modules/power/supermatter/supermatter.dm

```diff
@@ -127,6 +138,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	radio.keyslot = new radio_key
 	radio.listening = 0
 	radio.recalculateChannels()
+	distort = new(src)
+	vis_contents += distort
```

Comment:
Not really a fan of vis contents due to seemingly being far more maptick heavy than the alternative. Could this just be an overlay instead or does that mess with the rendering too much?